src/DGtal/base/Alias.h

```diff
@@ -199,61 +199,67 @@ user forward an Alias<T> parameter.
     inline ~Alias() {}
 
     /**
-       Constructor from const reference to an instance of T. Invalid.
+       Constructor from const reference to an instance of T. Deleted.
+
+       Aliasing a const-ref is an error. Consider ConstAlias instead.
     */
-    inline Alias( const T& ) : myParam( CONST_LEFT_VALUE_REF ), myPtr( 0 )
-    { ASSERT(( false && "[Alias::Alias( const T& )] Aliasing a const-ref is an error. Consider ConstAlias instead." )); }
+    Alias( const T& ) = delete;
```

Comment:
By the way, I was thinking about updating the templates in DGtalScripts to use those keywords and to allow move operations (in fact, declaring a copy operation prevents the compiler to automatically generate move operations).